calculator.js

```diff
@@ -1,5 +1,49 @@
-const exampleAdditionInput = {
-  num1: 3,
-  num2: 5,
-  operation: 'add',
-}
+// This line "loads" the prompt package and puts it into a variable we can use called prompt
+// Lines like this (that use require()) typically go on the top of the file
+const prompt = require("prompt");
+
+const calculateUserInput = function(error, promptInput) {
+    calculate(promptInput.num1, promptInput.num2, promptInput.operation);
```

Comment:
You will need to convert the input into integers or floats before doing the math.  Prompt reads things in as strings.

```suggestion
    calculate(parseInt(promptInput.num1), parseInt(promptInput.num2), promptInput.operation);
```